sites/feedback/index.json-ld

```diff
@@ -14,10 +14,10 @@
 		"en": "Documentation on how to use the elements of the feedback area.",
 		"fr": "Documentation sur l'utilisation des éléments de la zone commentaires."
 	},
-	"modified": "2023-01-10",
+	"modified": "2024-07-19",
 	"componentName": "feedback",
 	"status": "stable",
-	"version": "2.0",
+	"version": "2.1",
```

Comment:
Need a reality check on whether it was appropriate to update these lines.

The feedback component's version seems to be tracked separately from the PFT "sub-component"...

Since I updated the PFT to version 1.1 (shown as 2 in JSON-LD iteration variables)... I figured it'd also make sense to update feedback to 2.1 (since PFT "belongs" to it).